wiki/Gameplay/Fingerlock/de.md

```diff
@@ -0,0 +1,29 @@
+---
+stub: true
+tags:
+  - wrist lock
+  - wristlock
+  - wrist-lock
+  - finger-lock
+  - finger lock
+---
+
+# Fingerlock
+
+*Nicht zu verwechseln mit [Notelock](/wiki/Gameplay/Notelock) oder [Mind block](/wiki/Gameplay/Mind_block).*
```

Comment:
```suggestion
*Nicht zu verwechseln mit [Notelock](/wiki/Gameplay/Notelock) oder [Denkblockade](/wiki/Gameplay/Mind_block).*
```
TicClick asked if this PR is ready for merge. I took another look at it and the only thing I would change is `Mind block` to `Denkblockade` since this is used in the other article. Otherwise this PR is ready.